src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationServiceTest.java

```diff
@@ -386,11 +393,61 @@ void validateCurrentPositionCaseClosed(String state, boolean expected) {
     void shouldValidateJurisdictionCodesWithEmptyCodes() {
         List<String> errors = new ArrayList<>();
         caseDetails3.getCaseData().setJudgementCollection(new ArrayList<>());
-        eventValidationService.validateJurisdictionCodes(caseDetails3.getCaseData(), errors);
+        eventValidationService.validateJurisdiction(caseDetails3.getCaseData(), errors);
 
         assertEquals(0, errors.size());
     }
 
+    @Test
+    void shouldValidateDisposalDateInFuture() {
+        List<String> errors = new ArrayList<>();
+        eventValidationService.validateJurisdiction(setCaseDataForDisposalDateTest("2777-12-23","2022-01-20T00:00:00.000"), errors);
+        assertEquals(EventValidationService.DISPOSAL_DATE_IN_FUTURE, errors.get(0));
```

Comment:
An improved approach for this is:
```
assertThat(...)
  .isEqualTo(...);
```
Please apply to all cases in this PR

Reply:
done